contracts/Option.sol

```diff
@@ -2,31 +2,36 @@
 
 pragma solidity ^0.6.0;
 
-import "@nomiclabs/buidler/console.sol";
-
-import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
-import { SafeMath } from "@openzeppelin/contracts/math/SafeMath.sol";
-import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import { IWriterRegistry } from "./interface/IWriterRegistry.sol";
-import { Obligation } from "./Obligation.sol";
-import { IObligation } from "./interface/IObligation.sol";
-import { IOption } from "./interface/IOption.sol";
-import { European } from "./European.sol";
-import { IReferee } from "./interface/IReferee.sol";
-import { Bitcoin } from "./types/Bitcoin.sol";
+import '@nomiclabs/buidler/console.sol';
+
+import {Ownable} from '@openzeppelin/contracts/access/Ownable.sol';
+import {SafeMath} from '@openzeppelin/contracts/math/SafeMath.sol';
+import {IERC20} from '@openzeppelin/contracts/token/ERC20/IERC20.sol';
+import {IWriterRegistry} from './interface/IWriterRegistry.sol';
+import {Obligation} from './Obligation.sol';
+import {IObligation} from './interface/IObligation.sol';
+import {IOption} from './interface/IOption.sol';
+import {European} from './European.sol';
+import {IReferee} from './interface/IReferee.sol';
+import {Bitcoin} from './types/Bitcoin.sol';
 
 /// @title Option ERC20
 /// @author Interlay
 /// @notice Represents options that may be exercised for the
 /// backing currency in exchange for the underlying BTC.
 contract Option is IOption, IERC20, European, Ownable {
-    using SafeMath for uint;
-
-    string constant ERR_TRANSFER_EXCEEDS_BALANCE = "Amount exceeds balance";
-    string constant ERR_APPROVE_TO_ZERO_ADDRESS = "Approve to zero address";
-    string constant ERR_TRANSFER_TO_ZERO_ADDRESS = "Transfer to zero address";
-    string constant ERR_APPROVE_FROM_ZERO_ADDRESS = "Approve from zero address";
-    string constant ERR_TRANSFER_FROM_ZERO_ADDRESS = "Transfer from zero address";
+    using SafeMath for uint256;
+
+    string
+        internal constant ERR_TRANSFER_EXCEEDS_BALANCE = 'Amount exceeds balance';
```

Comment:
Am I the only one to find this style completely horrendous? :joy: 
I would rather go with the following but it's up to you!

```solidity
string internal constant
ERR_TRANSFER_EXCEEDS_BALANCE = 'Amount exceeds balance';
```

Reply:
This was a prettifier thing unfortunately, still need to find the correct setting...